client/src/locales/en-US.json

```
@@ -1722,7 +1722,12 @@
                 "description": "You can change your credit card at any time.",
                 "update": "Update"
             },
-            "processing": "Your order is being processed.",
+            "processing": {
+                "title": "Welcome to your customer area",
```

No use of the name variable?
```suggestion
                "title": "Welcome to your customer area {name}",
```

---

No, I removed the variable.